source/features/view-color.tsx

```diff
@@ -0,0 +1,166 @@
+import features from '../libs/features';
+
+function init(): void {
+	addColors();
+}
+const namedColors: string = [
```

Comment:
Can probably be replaced by some npm package, like [color-name-list](https://www.npmjs.com/package/color-name-list)...